src/host/alloc.c

```diff
@@ -65,6 +65,24 @@ void heap_trace(struct mm_heap *heap, int size)
 	malloc_info(0, stdout);
 }
 
+int memcpy_s(void *dest, size_t dest_size,
+	     const void *src, size_t src_size)
+{
+	if (!dest || !src)
```

Comment:
It would also be useful to place trace_error() in here too for debug (since any failure here will usually indicate a fatal error somewhere else).

Reply:
There is implementation of memcpy_s merged already and this one is host specific. Adding traces should be done for all of theme in separate pr.

Reply:
Removed that part from commit will create separate commit for traces in memcpy